cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java

```diff
@@ -124,7 +124,7 @@ public class TetheringAgentService extends AbstractRetryableScheduledService {
                         ProvisionerNotifier provisionerNotifier,
                         NamespaceQueryAdmin namespaceQueryAdmin) {
     super(RetryStrategies.fromConfiguration(cConf, "tethering.agent."));
-    this.connectionInterval = TimeUnit.SECONDS.toMillis(cConf.getLong(Constants.Tethering.CONNECTION_INTERVAL, 10L));
+    this.connectionInterval = TimeUnit.SECONDS.toMillis(cConf.getLong(Constants.Tethering.CONNECTION_INTERVAL, 1L));
```

Comment:
Generally we have the default value defined inside the `cdap-default.xml` instead of in the code so that the cdap-default.xml contains all the configurable options.

Reply:
fixed